daemon/cluster/executor/container/container.go

```diff
@@ -544,19 +544,6 @@ func (c *containerConfig) serviceConfig() *clustertypes.ServiceConfig {
 	return svcCfg
 }
 
-// networks returns a list of network names attached to the container. The
-// returned name can be used to lookup the corresponding network create
-// options.
-func (c *containerConfig) networks() []string {
```

Comment:
The `networks()` is returning the name only, which should not be encouraged (should always use ID if possible).